features/step_definitions/api-token.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const Assert = require('chai').assert;
+const jwt = require('jsonwebtoken');
+const request = require('../support/request');
+
+module.exports = function server() {
+    // eslint-disable-next-line new-cap
```

Comment:
Oh, huh, I could have sworn ESlint was complaining about it earlier. Apparently not!